autodiff/forward/eigen.hpp

```diff
@@ -165,4 +273,7 @@ auto jacobian(const Function& f, Wrt&& wrt, Args&& args) -> Eigen::MatrixXd
 
 } // namespace autodiff::forward
 
+namespace autodiff {
+using forward::wrtpack;
```

Comment:
We'll probably have just `wrt` in the future, when the usage of `wrt` to compute higher-order derivatives is modified. We may want to have such note in the tutorials.

Reply:
Ok, I'll do examples.